src/Meshes/geometry.ts

```diff
@@ -297,6 +296,11 @@ export class Geometry implements IGetSetVerticesData {
         if (this._vertexArrayObjects) {
             this._disposeVertexArrayObjects();
             this._vertexArrayObjects = {}; // Will trigger a rebuild of the VAO if supported
+
+            var meshes = this._meshes;
+            for (var index = 0; index < numOfMeshes; index++) {
+                meshes[index].invalidateInstanceVertexArrayObject();

```

Comment:
Here we only want to do this if instances are supported. I'm not sure how to do that so I need some help. This is also part of the pieces of Mesh that has been split off into instancedMesh.ts.

Reply:
We can simply cast it as any and test for invalidateInstanceVertexArrayObject

if ((meshes[index] as any).invalidateInstanceVertexArrayObject()) {
     (meshes[index] as InstanceMesh).invalidateInstanceVertexArrayObject();
}

Reply:
I tried doing it using an empty function in Mesh and then a specialized function in instancedMesh.ts. Just for now, lets see how it feels.